packages/std/src/os/aos_querier.rs

```diff
@@ -145,6 +149,14 @@ impl AOSQuerier {
         AOSQuerier::kernel_address_getter(querier, kernel_addr, ADO_DB_KEY)
     }
 
+    /// Queries the kernel's raw storage for the IBC Registry's address
+    pub fn ibc_registry_address_getter(
+        querier: &QuerierWrapper,
+        kernel_addr: &Addr,
+    ) -> Result<Addr, ContractError> {
+        AOSQuerier::kernel_address_getter(querier, kernel_addr, IBC_REGISTRY_KEY)
```

Comment:
nitpick: The naming is a bit confusing. You'd think that `kernel_address_getter` would get the kernel's address. Maybe `kernel_addresses_getter` would be better?